packages/account-v2/src/containers/POAForm/Status/Submitted/Submitted.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { useHistory } from 'react-router-dom';
+import { Button } from '@deriv-com/ui/dist/components/Button';
+import { Text } from '@deriv-com/ui/dist/components/Text';
+import IcPOAVerified from '../../../../assets/poa/ic-poa-verified.svg';
+import { isNavigationFromDerivGO, isNavigationFromP2P } from '../../../../utils/platform';
+import POAStatus from '../POAStatus';
+
+type TSubmitted = {
+    needsPOI: boolean;
+    redirectButton?: React.ReactNode;
+};
+
+export const Submitted = ({ needsPOI, redirectButton }: TSubmitted) => {
+    const history = useHistory();
+    const isRedirectedFromPlatform = isNavigationFromP2P() || isNavigationFromDerivGO();
+
+    let actionButton = null;
+
+    if (needsPOI) {
+        actionButton = (
+            <Button
+                onClick={() => {
+                    history.push('/account/proof-of-identity');
+                }}
+            >
+                Proof of identity
+            </Button>
+        );
+    } else if (redirectButton) {
+        actionButton = redirectButton;
+    } else if (!isRedirectedFromPlatform) {
+        actionButton = (
+            <Button
+                onClick={() => {
+                    history.push('/');
```

Comment:
Can we use constants from `@deriv/shared` ?

Reply:
created a route contants file.